### PR TITLE
Updated the README with mention of our tshirt giveaway

### PR DESCRIPTION
**Pull request by jwoertink**
Adding the giveaway notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +172,18 @@ You need to make sure to install the Crystal dependencies.
 
 > Run specific tests with `script/test <path_to_spec>`
 
+### Lucky to have you!
+
+We love all of the community members that have put in hard work helping to make Lucky better.
+If you're one of those people, we want to give you a t-shirt!
+
+Shirts are given to community members that have put in a ton of effort and contributions in to Lucky.
```
Comment (paulcsmith) [inline README.md:180]: I think this sentence can probably be removed since I think the first 2 cover it

File: README.md [hunk 2]
```diff
@@ -172,6 +172,18 @@ You need to make sure to install the Crystal dependencies.
 
 > Run specific tests with `script/test <path_to_spec>`
 
+### Lucky to have you!
+
+We love all of the community members that have put in hard work helping to make Lucky better.
```
Comment (paulcsmith) [inline README.md:177]:
```suggestion
We love all of the community members that have put in hard work to make Lucky better.
```

File: README.md [hunk 3]
```diff
@@ -172,6 +172,18 @@ You need to make sure to install the Crystal dependencies.
...
+If you're one of those people, we want to give you a t-shirt!
+
+Shirts are given to community members that have put in a ton of effort and contributions in to Lucky. 
+This includes things like submitting PRs with bug fixes and feature implementations, helping other members
```
Comment (paulcsmith) [inline README.md:181]:
```suggestion
To get a shirt we ask that you have made a significant contribution to Lucky. This includes things like submitting PRs with bug fixes and feature implementations, helping other members
```

**Comment by jwoertink** [comment]
Thanks for the suggestions!

**Pull request merged**